compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/SkiaParagraph.skiko.kt

```diff
@@ -492,11 +493,34 @@ internal class ParagraphBuilder(
         return null
     }
 
+    private fun makeSkFontRasterizationSettings(style: TextStyle): SkFontRastrSettings {
+        val rasterizationSettings = style.paragraphStyle.platformStyle?.fontRasterizationSettings
+            ?: FontRasterizationSettings.defaultForCurrentPlatform
+        val edging = when (rasterizationSettings.edging) {
+            FontEdging.ALIAS -> org.jetbrains.skia.FontEdging.ALIAS
+            FontEdging.ANTI_ALIAS -> org.jetbrains.skia.FontEdging.ANTI_ALIAS
+            FontEdging.SUBPIXEL_ANTI_ALIAS -> org.jetbrains.skia.FontEdging.SUBPIXEL_ANTI_ALIAS
+        }
+        val hinting = when (rasterizationSettings.hinting) {
+            FontHinting.NONE -> org.jetbrains.skia.FontHinting.NONE
+            FontHinting.SLIGHT -> org.jetbrains.skia.FontHinting.SLIGHT
+            FontHinting.NORMAL -> org.jetbrains.skia.FontHinting.NORMAL
+            FontHinting.FULL -> org.jetbrains.skia.FontHinting.FULL
+        }
+        return SkFontRastrSettings(
+            edging = edging,
+            hinting = hinting,
+            subpixel = rasterizationSettings.subpixelPositioning
+            // rasterizationSettings.autoHintingForced is ignored here for now because it's not supported in skia
+        )
```

Comment:
```suggestion
        val edging = when (rasterizationSettings.edging) {
            FontSmoothing.None-> org.jetbrains.skia.FontEdging.ALIAS
            FontSmoothing.AntiAlias -> org.jetbrains.skia.FontEdging.ANTI_ALIAS
            FontSmoothing.SubpixelAntiAlias -> org.jetbrains.skia.FontEdging.SUBPIXEL_ANTI_ALIAS
        }
        val hinting = when (rasterizationSettings.hinting) {
            FontHinting.None -> org.jetbrains.skia.FontHinting.NONE
            FontHinting.Slight -> org.jetbrains.skia.FontHinting.SLIGHT
            FontHinting.Normal -> org.jetbrains.skia.FontHinting.NORMAL
            FontHinting.Full -> org.jetbrains.skia.FontHinting.FULL
        }
        return SkFontRastrSettings(
            edgint = edging,
            hinting = hinting,
            subpixel = rasterizationSettings.subpixelPositioning
            // rasterizationSettings.autoHintingForced is ignored here for now because it's not supported in skia
        )
```